<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactItemWidget</class>
 <widget class="QWidget" name="ContactItemWidget">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Global*/
QWidget{
	background-color:transparent;
}

/*QPushButton*/
QPushButton{
	background-color:transparent;
	border:none;
	padding: 0 0;
	margin:0;
}

/*QMenu*/
QMenu{
	background-color:white;
	padding: 1;
	margin: 0;
}
QMenu::item{
	padding: 10px 10px;
}
QMenu::item::selected{
	color:white;
	background-color:#09f;
}
QMenu::item::pressed{
	color:white;
	background-color:#07f;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QIconButton" name="messenger_bt">
     <property name="styleSheet">
      <string notr="true">/*QPushButton{
	image: url(&quot;:/icons/messenger_normal&quot;);
 }
QPushButton:hover{
	image: url(&quot;:/icons/messenger_hover&quot;);
 }
QPushButton:pressed{
	image: url(&quot;:/icons/messenger_pressed&quot;);
 }*/</string>
     </property>
     <property name="icon">
      <iconset resource="../res/res.qrc">
       <normaloff>:/icons/messenger_normal</normaloff>
       <activeoff>:/icons/messenger_hover</activeoff>
       <selectedoff>:/icons/messenger_pressed</selectedoff>:/icons/messenger_normal</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="call_bt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(&quot;:/icons/call_normal&quot;);
 }
QPushButton:hover{
	image: url(&quot;:/icons/call_hover&quot;);
 }
QPushButton:pressed{
	image: url(&quot;:/icons/call_pressed&quot;);
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="options_bt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(&quot;:/icons/options_normal&quot;);
 }
QPushButton:hover{
	image: url(&quot;:/icons/options_hover&quot;);
 }
QPushButton:pressed{
	image: url(&quot;:/icons/options_pressed&quot;);
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QIconButton</class>
   <extends>QPushButton</extends>
   <header>qiconbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
